README.md

```diff
@@ -1,3 +1,83 @@
 # Oragono Docker
 
-This repo holds Oragono's Dockerfiles and such.
+This repo holds [Oragono](https://github.com/oragono/oragono)'s Dockerfiles and
+related materials.
+
+## Tags
+
+The following tags are available:
+
+ * `dev` - latest development version; published periodically (may be unstable)
+
+## Quick start
+
+The Oragono docker image is designed to work out of the box - it comes with a
+usable default config and will automatically generate self-signed TLS
+certificates. To get a working ircd, all you need to do is run the image and
+expose the ports:
+
+```shell
+docker run -d -P oragono/oragono:tag
+```
+
+This will start Oragono and listen on ports 6667 (plain text) and 6697 (TLS).
```

Comment:
Hmm, should we mention here that the default operator (admin user) details are created at runtime and dumped to the terminal? Might be good to note it here, just so that users know to keep an eye out for those details

Reply:
Yeah we can do this afterwards as well